08_deploy_bmo.sh

```diff
@@ -13,8 +13,24 @@ cp -r $BMOPATH/deploy ocp/.
 sed -i 's/namespace: .*/namespace: openshift-machine-api/g' ocp/deploy/role_binding.yaml
 
 # Start deploying on the new cluster
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/service_account.yaml --namespace=openshift-machine-api
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/role.yaml --namespace=openshift-machine-api
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/role_binding.yaml
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/operator.yaml --namespace=openshift-machine-api
-oc --config ocp/auth/kubeconfig apply -f ocp/deploy/crds/metalkube_v1alpha1_baremetalhost_crd.yaml
+KUBECONFIG=ocp/auth/kubeconfig
+oc --as system:admin apply -f $BMOPATH/deploy/service_account.yaml
+oc --as system:admin apply -f $BMOPATH/deploy/role.yaml
+oc --as system:admin apply -f $BMOPATH/deploy/role_binding.yaml
+oc --as system:admin apply -f $BMOPATH/deploy/crds/metalkube_v1alpha1_baremetalhost_crd.yaml
+oc --as system:admin apply -f $BMOPATH/deploy/operator.yaml
+
+# Give the operator a chance to quiesce. Ideas to check instead of sleep?
+sleep 10
+
+oc --as system:admin proxy &
+proxy_pid=$!
+
+for node in $(oc --as system:admin get nodes -o template --template='{{range .items}}{{.metadata.uid}}:{{.metadata.name}}{{"\n"}}{{end}}'); do
+    name=$(echo $node | cut -f2 -d':')
+    uid=$(echo $node | cut -f1 -d':')
+    addresses=$(oc get node $name -o json | jq -c '.status.addresses')
+    curl -X PATCH http://localhost:8001/apis/machine.openshift.io/v1beta1/namespaces/openshift-machine-api/machines/$CLUSTER_NAME-$name/status -H "Content-type: application/merge-patch+json" -d '{"status":{"addresses":'"${addresses}"',"nodeRef":{"kind":"Node","name":"'"${name}"'","uid":"'"${uid}"'"}}}'
```

Comment:
I believe this will work for the 3 Machines created for masters, but the name for the Machines for workers will include a random generated string when they get created by a MachineSet.  I'm guessing your cluster was launched before enough fixes got in to where worker Machines got created (probably just yesterday).